src/openms/source/FORMAT/IdXMLFile.cpp

```diff
@@ -603,7 +603,7 @@ namespace OpenMS
       // sequence
       String tmp;
       optionalAttributeAsString_(tmp, attributes, "sequence");
-      prot_hit_.setSequence(std::move(tmp));
+      prot_hit_.setSequence(tmp);
```

Comment:
also here

Reply:
I guess there is no `void setSequence(const String && sequence);` but the solution is to implement this

https://github.com/OpenMS/OpenMS/blob/8594c2ad50b1e3d3263235ea592ed02049dbcb11/src/openms/include/OpenMS/METADATA/ProteinHit.h#L189

Reply:
you mean void setSequence(String && sequence) ?

Reply:
will address this is in #5685